README.md

```diff
@@ -1,2 +1,269 @@
 # pyvaultone
-Project to abstract vault one interaction.
+
+[![Build Status](https://travis-ci.com/NeowayLabs/pyvaultone.svg?branch=main)](https://travis-ci.com/NeowayLabs/pyvaultone)
+
+This project is a library to abstract relevant interactions to the VaultOne application API. Relevant interactions are defined as necessary interactions in order for a bot to retrieve credentials that will be used in the crawling process.
```

Comment:
maybe is good to put here one link to Vaultone application..  @vavieira10

Reply:
@LeoCBS Done!